src/hardware/openbench-logic-sniffer/protocol.h

```diff
@@ -37,7 +37,6 @@
 #define CMD_RESET                  0x00
 #define CMD_RUN                    0x01
 #define CMD_ID                     0x02
-#define CMD_TESTMODE               0x03
```

Comment:
Yes, it can go. I think it is historically interesting, but then again, it is already in the repo.